katalog/curator-s3/elasticsearch/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM elasticsearch:6.4.1
+
+RUN elasticsearch-plugin install repository-s3 -b
+
+RUN echo -Des.allow_insecure_settings=true >> /usr/share/elasticsearch/config/jvm.options
```

Comment:
Why is this config needed?

Reply:
for this reason: 
```     
     initContainers:
            - name: curator-add-task
              image: quay.io/sighup/kubectl:v1.13.2
              command:
                - /bin/sh
                - -c
                - 'kubectl create job curator-add-task-$(date +%s) --from=cronjob/snapshot-configurator'
              envFrom:
                - secretRef:
                    name: curator-aws
                    ```

Reply:
I mean `allow_insecure_settings`

Reply:
https://github.com/elastic/elasticsearch/issues/26589

Reply:
it is a 2017 thread. Is it still relevant?

Reply:
seems still relevant because of the static credentials, so yes it is

Reply:
an alternative of making a new elasticsearch docker image, could be an initContainer patch in the elasticsearch setup like this:

```yaml
        initContainers:
        - name: install-plugins
          command:
          - sh
          - -c
          - |
            bin/elasticsearch-plugin install --batch repository-s3 && echo -Des.allow_insecure_settings=true >> /usr/share/elasticsearch/config/jvm.options
```

Which is preferred solution

Reply:
I'm not sure to have a strong opinion here

Reply:
Do we need to serve this "container image" (after build) to use it?
Why does it have ES version 6 while we are already on ES 7?

Reply:
So the Dockerfile could be placed in the fury-images repository no?

Reply:
we should remove it and that's it. I already use this kind of setup in our customer and I skipped the custom docker image, using the initContainer and some patch. Is up to you, IMHO, is not necessary to maintain a docker image for this.

Reply:
So, remove it or move it to the examples dir :)